.github/workflows/codeql-analysis.yml

```diff
@@ -13,13 +13,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-name: "CodeQL Java"
+name: "CodeQL"
 
 on:
   push:
-    branches: [ master ]
+    branches-ignore:
+      - 'dependabot/**'
```

Comment:
Can you add a [`concurrency` setting](https://docs.github.com/en/actions/how-tos/write-workflows/choose-when-workflows-run/control-workflow-concurrency) to prevent workflows to run twice on internal branches of this repo?

Reply:
See the latest commit https://github.com/apache/commons-daemon/pull/272/commits/f47ae6d062b43bff7330259de024b1610e1e0a3f